src/main/flight/gps_rescue.c

```diff
@@ -152,6 +152,7 @@ float         averageThrottle = 0.0;
 float         altitudeError = 0.0;
 uint32_t      throttleSamples = 0;
 bool          magForceDisable = false;
+uint16_t      descentDistance;
```

Comment:
This does not need to be global, it should be a static inside `updateGPSRescueState()`

Reply:
ok